src/components/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -30,142 +32,156 @@ export interface AmbassadorCardProps {
   ambassador: AmbassadorType;
   onClose?: () => void;
   className?: string;
+  disableCardEffects?: boolean;
 }
 
 export default function AmbassadorCard(props: AmbassadorCardProps) {
-  const { ambassadorKey, ambassador, onClose, className } = props;
+  const { ambassadorKey, ambassador, onClose, className, disableCardEffects } =
+    props;
   const images = getAmbassadorImages(ambassadorKey);
   const mod =
     window?.Twitch?.ext?.viewer?.role === "broadcaster" ||
     window?.Twitch?.ext?.viewer?.role === "moderator";
+  const glareOpacity = disableCardEffects ? 0.0 : 0.5;
 
   return (
-    <div
-      className={classes(
-        styles.ambassadorCard,
-        className,
-        ambassador.birth && isBirthday(ambassador.birth) && styles.birthday,
-      )}
+    <Tilt
+      tiltEnable={!disableCardEffects}
+      glareEnable={!disableCardEffects}
+      glareMaxOpacity={glareOpacity}
+      glareBorderRadius="1rem"
+      glarePosition="bottom"
+      scale={1.0}
+      perspective={5000}
+      className={classes(styles.ambassadorCard, className)}
```

Comment:
Unfortunately something here sets a transform-origin that makes this perspective shift look rather funky. I wonder if we need to move that transform-origin (which is used for the slide-in/out animation) into a wrapper around the Tilt component?

Reply:
Ah I wasn't sure what was happening here, thanks for the spot - I'll take a look!

Reply:
When investigating this, I trialled removing the transform-origin from the ambassador card class within the overlay scss module and this appears to achieve the desired result. The cards still slide in from left and now the tilt works correctly. Let me know what you think, as far as I could tell, removing the transform-origin hasn't had any unwanted side effects!